src/components/Credentials.js

```diff
@@ -4,7 +4,7 @@ import { Link } from "react-router-dom";
 
 // description and examples in `asab-webui/doc/credentials.md`
 
-export function Credentials(props) {
+export function Credentials({ ...props }) {
```

Comment:
@Aringoaway Will the functionality of the component remains the same? I mean - are `props` passed in the `Credentials` component really passed to it (e.g. props for `props.app.axiosCreate(...)`)?

Reply:
@Pe5h4 Yes, it really passed in the Credentilas

Reply:
@Aringoaway ok